tests/e2e/creation_test.go

```diff
@@ -444,6 +451,54 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testTrustedCABundle() error {
+	CAConfigMapName := "odh-trusted-ca-bundle"
+	CADataFieldName := "odh-ca-bundle.crt"
+
+	if tc.testDSCI.Spec.TrustedCABundle.ManagementState == "Managed" {
+		istrustedCABundleUpdated, err := trustedcabundle.IsTrustedCABundleUpdated(tc.ctx, tc.customClient, tc.testDSCI)
+
+		if istrustedCABundleUpdated == true {
+			return fmt.Errorf("odh-trusted-ca-bundle in config map does not match with DSCI's TrustedCABundle.CustomCABundle, needs update: %w", err)
+		}
+
+		err = trustedcabundle.AddCABundleCMInAllNamespaces(tc.ctx, tc.customClient, logr.Logger{}, tc.testDSCI)
```

Comment:
Why are we explicitly adding and removing the configmap from all namespaces? Shouldn't this be created by the controller?